client/src/lib/activity.ts

```diff
@@ -32,7 +36,7 @@ export function activityFallsOnDay(activity: ActivityWithIds, date: Datelike) {
 export function activityDuration(activity: ActivityWithIds) {
 	const [start, end] = [activityStart(activity), activityEnd(activity)];
 
-	return Math.min(end.diff(start, "minute") / 60, 24);
+	return Math.min(end.diff(start, "minute") / 60, 24); // TODO: the maximum duration should be (24 - startHour) so that the activity doesn't overflow the day
```

Comment:
In this PR, I implemented this in `useToday()` (I think), so this TODO can be removed.